### DISCUSSION TITLE
Track nested-deps benchmark over time

### DISCUSSION THREAD

**Discussion by timneutkens** [vercel/next.js#62371] [Ideas]
Currently there is a benchmark for 3K/10K components available here: https://github.com/vercel/next.js/tree/canary/bench/nested-deps. We want to make sure these are measured over time to see  how improvements affect performance.

Bench: https://github.com/vercel/next.js/tree/canary/bench/nested-deps

Yarn has some prior art here where they compare each package manager and send the data to datadog:
Actions workflow: https://github.com/yarnpkg/berry/blob/ff39e53e3622d197764603c73e2e98b3f2ddc586/.github/workflows/benchmark-workflow.yml
Data sending script: https://github.com/yarnpkg/berry/blob/ff39e53e3622d197764603c73e2e98b3f2ddc586/scripts/submit-bench-data.js

**Comment by samcx** [comment]
Moving this to <picture data-single-emoji=":nextjs:" title=":nextjs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/nextjs/2a85d633ae594556.png" alt=":nextjs:" align="absmiddle"></picture> [discussions](https://github.com/vercel/next.js/discussions) since this is a feature request!
